lgc/patch/PatchResourceCollect.cpp

```diff
@@ -1328,7 +1320,7 @@ void PatchResourceCollect::matchGenericInOut() {
   for (unsigned idx = 0; idx < 2; ++idx) {
```

Comment:
Unroll this loop and I think the code becomes a bit shorter and certainly simpler.